pkg/operators/wasm/maps.go

```diff
@@ -195,6 +201,16 @@ func (i *wasmOperatorInstance) mapLookup(ctx context.Context, m wapi.Module, sta
 	stack[0] = 0
 }
 
+func mapAcceptsFD(m *ebpf.Map) bool {
+	mapType := m.Type()
+	for _, typ := range mapTypesAcceptingFD {
+		if mapType == typ {
+			return true
+		}
+	}
+	return false
```

Comment:
```suggestion
	return slices.Contains(mapTypesAcceptingFD, m.Type())
```

Reply:
Given that it's heavily used (one invocation for each map update), I'm wondering if it makes sense to implement by using a switch statement instead.